CefSharp.Example/CefExample.cs

```diff
@@ -24,6 +24,7 @@ public static class CefExample
         public const string BasicSchemeTestUrl = "custom://cefsharp/SchemeTest.html";
         public const string ResponseFilterTestUrl = "custom://cefsharp/ResponseFilterTest.html";
         public const string DraggableRegionTestUrl = "custom://cefsharp/DraggableRegionTest.html";
+        public const string DownloadHandlerTestUrl = "custom://cefsharp/DownloadHandlerTest.html";
```

Comment:
Sorry, meant to revert this. I had added a new test html file but it didn't turn out to be a very good addition.